tests/telemetry/test_telemetry.py

```diff
@@ -60,3 +76,27 @@ def test_telemetry_enabledbydefault(duthost):
         if str(k) == "status":
             status_expected = "enabled";
             pytest_assert(str(v) == status_expected, "Telemetry feature is not enabled")
+
+def test_telemetry_ouput(duthost, ptfhost):
+    """Run pyclient from ptfdocker and show gnmi server outputself.
+       For pyclient to work client_auth need to be set to false.
+    """
+    set_client_auth = duthost.shell('/usr/bin/redis-cli -n 4 hset "TELEMETRY|gnmi" "client_auth" "false"', module_ignore_errors=False)
```

Comment:
move this to a function and comment on its purpose.

Reply:
better to this as a fixture.

Reply:
put client_auth and restart telemtry as fixture, then yield.

Reply:
you also need to wait till the telemtry docker is full restarted before run the test because if the telemetry server is not fully restart, you may have connection error.

Reply:
updated based on this conversation. Please review